Purchases/Purchasing/Purchases.swift

```diff
@@ -1160,6 +1169,50 @@ public extension Purchases {
     }
 #endif
 
+    /**
+     * Use this method to find eligibility for this user for
+     * [iOS Promotional Offers](https://docs.revenuecat.com/docs/ios-subscription-offers#promotional-offers).
+     * - Note: If you're looking to use free trials or Introductory Offers instead,
+     * use ``Purchases/checkTrialOrIntroductoryPriceEligibility(_:completion:)``.
+     *
+     * - Parameter discount: The `StoreProductDiscount` to apply to the product.
+     * - Parameter product: The `StoreProduct` the user intends to purchase.
+     * - Parameter completion: A completion block that is called when the `PromotionalOfferEligibility` is returned.
+     * If it was not successful, there will be an `Error`.
+     */
+    @available(iOS 12.2, macOS 10.14.4, macCatalyst 13.0, tvOS 12.2, watchOS 6.2, *)
+    @objc(checkPromotionalOfferEligibility:product:completion:)
+    func checkPromotionalOfferEligibility(forProductDiscount discount: StoreProductDiscount,
+                                          product: StoreProduct,
+                                          completion: @escaping (PromotionalOfferEligibility, Error?) -> Void) {
+        guard let sk1Product = product.sk1Product else {
+            // todo: add support for SK2 discounts
+            fatalError("StoreKit2 not supported yet")
+        }
+
+        purchasesOrchestrator.promotionalOffer(forProductDiscount: discount,
+                                               product: sk1Product) { promotionalOffer, error in
+            completion(promotionalOffer == nil ? .ineligible : .eligible, error)
+        }
+    }
+
+    /**
+     * Use this method to find eligibility for this user for
+     * [iOS Promotional Offers](https://docs.revenuecat.com/docs/ios-subscription-offers#promotional-offers).
+     * - Note: If you're looking to use free trials or Introductory Offers instead,
+     * use ``Purchases/checkTrialOrIntroductoryPriceEligibility(_:completion:)``.
+     *
+     * - Parameter discount: The `StoreProductDiscount` to apply to the product.
+     * - Parameter product: The `StoreProduct` the user intends to purchase.
+     * - Parameter completion: A completion block that is called when the `PromotionalOfferEligibility` is returned.
```

Comment:
Docc: use double backticks to point to the type, i.e.: \`\`PromotionalOfferEligibility\`\`

Reply:
same for the other types in this docstring, \`\`StoreProduct\`\`, \`\`StoreProductDiscount\`\`
And for the docstring in the non-async/await version of the method